src/stats/show_stats.py

```diff
@@ -113,12 +131,24 @@ def print_percentages_above_graph_bars(graph, data):
         i += 1
 
 
-async def send_graph_to_channel(ctx, stat_id, stat_title):
+async def send_graph_to_channel(ctx, url, stat_title):
     graph_embed = discord.Embed(title=stat_title, colour=discord.Colour(0xFFFF00))
-    graph_embed.set_image(url=upload_graph_to_imgur(stat_id))
+    graph_embed.set_image(url=url)
     await ctx.channel.send(embed=graph_embed)
 
 
-def upload_graph_to_imgur(stat_id):
-    uploaded_image = imgur.upload_image(f"{stat_id}.png")
-    return uploaded_image.link
+async def upload_graph_to_imgur_or_none(ctx, stat_id):
+    try:
+        uploaded_image = imgur.upload_image(f"{stat_id}.png")
```

Comment:
How come we can store the `{stat_id}.png` file now? Or is it just in memory? 🤷

Reply:
Storing files works only for short periods of time. 
https://help.heroku.com/K1PPS2WM/why-are-my-file-uploads-missing-deleted